lib/testscript_engine/testscript_runnable.rb

```diff
@@ -170,6 +175,20 @@ def load_fixtures
     end
   end
 
+  def load_profiles
+    script.profile.each do |profile|
+      next warning(:no_static_profile_id) unless profile.id
+      next warning(:no_static_profile_reference) unless profile.reference
+
+      profile_server = FHIR::Client.new("")
+      response = profile_server.send(:get, profile.reference, { 'Content-Type' => 'json' })
+      next if response.response[:code] == "200"
```

Comment:
Maybe broaden this a bit to any `2xx` response? Not just necessarily `200`

Reply:
Could just use something like `.starts_with?('2')` if `:code` is guaranteed to be a string

Reply:
> Could just use something like `.starts_with?('2')` if `:code` is guaranteed to be a string

Good suggestion. Done.